soc/nordic/common/mpu_regions.c

```diff
@@ -32,6 +36,10 @@ static struct arm_mpu_region mpu_regions[] = {
 	MPU_REGION_ENTRY("CAN120_MCAN", CAN120_BASE,
 			 REGION_RAM_NOCACHE_ATTR(CAN120_BASE, CAN120_SIZE)),
 #endif
+#if DT_NODE_HAS_STATUS(DT_NODELABEL(can121), okay)
+	MPU_REGION_ENTRY("CAN121_MCAN", CAN121_BASE,
```

Comment:
I can't find the MCAN addresses in IPS, but from the bus configuration, it seems possible to cover both 120 and 121 instances with a single MPU entry spanning both. What do you think?

Reply:
It is indeed possible to combine them, but I think it's OK to do this later.